resources/schemas/dbscripts/postgresql/cds-20.005-20.006.sql

```diff
@@ -0,0 +1,7 @@
+-- add target_cell columns back since data for target_cell should be coming from nab and nabmab assays, and not nabantigen metadata table
+ALTER TABLE cds.import_nab ADD COLUMN target_cell VARCHAR(250);
+ALTER TABLE cds.import_nabmab ADD COLUMN target_cell VARCHAR(250);
+
+-- Drop NOT NULL constraint since target_cell column is now expected to have null vals in nabantigen metadata table
+ALTER TABLE cds.import_nabantigen ALTER COLUMN target_cell DROP NOT NULL;
```

Comment:
Since target_cell is from assay itself, should it be removed from nabantigen? Or is it used somewhere else?

Reply:
Drienna had mentioned that "Values for target cell in the metadata table at this point should be blank." -- not too sure if there will be data in the future, hence have kept it as is for now.

Reply:
target_cell now removed, OK'd by client.